src/deepsparse/loggers/metric_functions/natural_language_processing/built_ins.py

```diff
@@ -0,0 +1,36 @@
+# Copyright (c) 2021 - present / Neuralmagic, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Set of functions for logging metrics from the natural language processing pipelines
+"""
+from typing import Dict, List, Union
+
+
+__all__ = ["sequence_length", "percent_unknown_tokens"]
+
+
+def sequence_length(sequence: Union[List[str], str]) -> Union[Dict[str, int], int]:
```

Comment:
`sequence_length` in NLP has the specific connotation of the number of tokens in a sequence rather than the string length. let's rename this to `string_length`